src/commandsAndMenu.tsx

```diff
@@ -558,6 +562,34 @@ export function addCommands(
           shell.add(diffWidget, 'main');
           shell.activateById(diffWidget.id);
 
+          // Search for the tab
+          const dockPanel = (app.shell as any)._dockPanel as DockPanel;
+
+          let tabPosition = -1;
+          const tabBar = find(dockPanel.tabBars(), bar => {
+            tabPosition = bar.titles.indexOf(diffWidget.title);
+            console.log('2', tabPosition);
+            return tabPosition !== -1;
+          });
+
+          // Get the tab
+          setTimeout(() => {
```

Comment:
Let's add some explanation about the need for `setTimeout`

```suggestion
          // We need to wait for the tab node to be inserted in the DOM
          setTimeout(() => {
```